RecyclerView/app/src/main/java/net/boondockradio/recyclerview/StatesDelegate.java

```diff
@@ -0,0 +1,6 @@
+package net.boondockradio.recyclerview;
+
+public interface StatesDelegate {
```

Comment:
normally we call this a `Listener` or `Callback`

`StateAdapterListener` or `StateAdapterCallback`